docs/topics/optimizing_memory_usage.adoc

```diff
@@ -4,6 +4,9 @@
 :context: optimizing_memory_usage
 
 include::modules/reviewing_resource_information.adoc[leveloffset=+1]
-include::modules/monitor_oso_quota_usage.adoc[leveloffset=+1]
+
+//include::modules/monitor_oso_quota_usage.adoc[leveloffset=+1]
+
 include::modules/reduce_hello_world_memory_usage.adoc[leveloffset=+1]
-include::modules/commit_push_changes_git.adoc[leveloffset=+1]
+
+//include::modules/commit_push_changes_git.adoc[leveloffset=+1]
```

Comment:
I don't agree with this removal. It's true that the module is reused, but that's the purpose of modules -- we don't want to force the user to jump around in the doc, going from link to link. Instead, we want to give them a clear path (A -> B)  for each task they're performing. If several tasks involve the same step, so be it.

Also, while identical, the module is being used in different contexts. If we were to link from here to a different context, it would be confusing.

Reply:
+1 for reuse. Modules+conditions make reuse easy and reuse is one of the main reasons we break up tasks into smaller tasks.